File: Prueba y error/ejercicios de semana santa php/Four php/Library/LicenciaB.php
<?php
require_once 'Licencia.php';

class LicenciaB extends Licencia {
    public function __construct() {
        $this->tipo = "Licencia B (Vehículos particulares)";
        $this->duracion = 18;
        $this->costo = 250000;
    }
}

File: Ejercicios de Semana Santa/ejercicios de semana santa java/Four java/Library/LicenciaB.java
package Library;

public class LicenciaB extends Licencia {
    public LicenciaB() {
        this.tipo = "B";
        this.costo = 1200000;
        this.duracion = 8;
    }

    public void mostrarDetalles() {
        System.out.println("\n--- Detalles de la Licencia ---");
        System.out.println("Tipo: " + tipo + " (Vehículos particulares)");
        System.out.println("Costo: " + String.format("%,.2f COP", costo));
        System.out.println("Duración: " + duracion + " meses");
    }
}
